moj_analytics/auth0_client.py

```diff
@@ -137,12 +150,47 @@ def update(self, resource):
     def get_all(self, resource_class):
         endpoint = "{}s".format(resource_class.__name__.lower())
 
-        resources = self.request("GET", endpoint)
+        items = []
+        total = None
+
+        params = {}
+        if self.use_pagination:
+            params = {
+                "include_totals": "true",
+                "page": 0,
+                "per_page": 50,
```

Comment:
Out of interest, how did you arrive at this figure..? It might be more efficient to ask for pages of `n` size where `n` is something else.

Reply:
I've just used the maximum the API allows to reduce the number of requests and kept it simple: https://auth0.com/docs/product-lifecycle/deprecations-and-migrations/migrate-to-paginated-queries

To be fair I could move this into a constant but I thought it was overkill...I'll tell you what, I'll move to a constant and add a comment saying this is the max the API allows to document it 👍

Reply:
Heh... 50 just sounded arbitrary. I agree that the max number of items in a page allowed by the API is definitely the way to go.